### PR TITLE
MAUI-860107: Updated UG content for Maui ImageEditor

### PR DESCRIPTION
**Pull request by Shalini-Ashokan**
**Description**
Updated UG content for image editor control.
**Topics**.
Getting started
Known issue
Double tap to activate editing.

### CODE REVIEW

File: MAUI/ImageEditor/getting-started.md [hunk 1]
```diff
@@ -307,4 +307,98 @@ The image editor utilizes the `AspectFit` image scaling of Image control to fit
 {% endhighlight %}
 {% endtabs %}
 
-N> The size value will only be available after the image has been loaded into view.
\ No newline at end of file
+N> The size value will only be available after the image has been loaded into view.
+
+## ImageEditor inside stack layout
+
+### VerticalStackLayout
```
Comment (karthikraja-arumugam) [inline MAUI/ImageEditor/getting-started.md:314]: #### VerticalStackLayout - use 4#

File: MAUI/ImageEditor/getting-started.md [hunk 2]
```diff
@@ -307,4 +307,98 @@ The image editor utilizes the `AspectFit` image scaling of Image control to fit
...
 
-N> The size value will only be available after the image has been loaded into view.
\ No newline at end of file
+N> The size value will only be available after the image has been loaded into view.
+
+## ImageEditor inside stack layout
+
+### VerticalStackLayout
+
+The image editor is placed inside vertical stack layout, and users can specify a minimum height request, with the default value already predefined as 100.
```
Comment (karthikraja-arumugam) [inline MAUI/ImageEditor/getting-started.md:316]: When the image editor is placed inside a vertical stack layout, users must define the required MinimumHeightRequest value. By default, this MinimumHeightRequest is set to 100.

File: MAUI/ImageEditor/getting-started.md [hunk 3]
```diff
@@ -307,4 +307,98 @@ The image editor utilizes the `AspectFit` image scaling of Image control to fit
...
+    verticalLayout.Add(imageEditor);
+    this.Content = verticalLayout;
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+### HorizontalStackLayout
+
+The image editor is placed inside horizontal stack layout, and users can specify a minimum width request, with the default value already predefined as 100.
```
Comment (karthikraja-arumugam) [inline MAUI/ImageEditor/getting-started.md:347]: When the image editor is placed inside a horizontal stack layout, users must define the required `MinimumWidthRequest` value. By default, this `MinimumWidthRequest`  is set to 100.

File: MAUI/ImageEditor/getting-started.md [hunk 4]
```diff
@@ -307,4 +307,98 @@ The image editor utilizes the `AspectFit` image scaling of Image control to fit
...
+    horizontalLayout.Add(imageEditor);
+    this.Content = horizontalLayout;
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+### Image is edited or not.
+
+The `IsImageEdited` property is used to determine whether any editing action has been performed on the image.
```
Comment (karthikraja-arumugam) [inline MAUI/ImageEditor/getting-started.md:379]: `IsImageEdited` - add link

File: MAUI/ImageEditor/getting-started.md [hunk 5]
```diff
@@ -307,4 +307,98 @@ The image editor utilizes the `AspectFit` image scaling of Image control to fit
...
+    </Grid>  
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+private void OnIsImageEditedClicked(object sender, EventArgs e)
+{
+    if (this.imageEditor.IsImageEdited)
+    {
+        this.imageEditor.ImageEffect(ImageEffect.Brightness, -0.6);
```
Comment (karthikraja-arumugam) [inline MAUI/ImageEditor/getting-started.md:399]: call save method

this.imageEditor.Save();

File: MAUI/ImageEditor/getting-started.md (hunk 1 above)
Comment (Shalini-Ashokan) [reply MAUI/ImageEditor/getting-started.md:314]: Used four #

File: MAUI/ImageEditor/getting-started.md (hunk 2 above)
Comment (Shalini-Ashokan) [reply MAUI/ImageEditor/getting-started.md:316]: Updated

File: MAUI/ImageEditor/getting-started.md (hunk 3 above)
Comment (Shalini-Ashokan) [reply MAUI/ImageEditor/getting-started.md:347]: Updated

File: MAUI/ImageEditor/getting-started.md (hunk 4 above)
Comment (Shalini-Ashokan) [reply MAUI/ImageEditor/getting-started.md:379]: Added link

File: MAUI/ImageEditor/getting-started.md (hunk 5 above)
Comment (Shalini-Ashokan) [reply MAUI/ImageEditor/getting-started.md:399]: Called save method

File: MAUI/ImageEditor/getting-started.md [hunk 6]
```diff
@@ -307,4 +307,97 @@ The image editor utilizes the `AspectFit` image scaling of Image control to fit
 {% endhighlight %}
 {% endtabs %}
 
-N> The size value will only be available after the image has been loaded into view.
\ No newline at end of file
+N> The size value will only be available after the image has been loaded into view.
+
+## ImageEditor inside stack layout
+
+#### VerticalStackLayout
```
Comment (karthikraja-arumugam) [inline MAUI/ImageEditor/getting-started.md:314]: Vertical StackLayout

File: MAUI/ImageEditor/getting-started.md [hunk 7]
```diff
@@ -307,4 +307,97 @@ The image editor utilizes the `AspectFit` image scaling of Image control to fit
...
+    imageEditor.Source = ImageSource.FromResource("MyProject.Resources.Images.image.jpeg");
+    imageEditor.MinimumHeightRequest = 400;
+    verticalLayout.Add(imageEditor);
+    this.Content = verticalLayout;
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+#### HorizontalStackLayout
```
Comment (karthikraja-arumugam) [inline MAUI/ImageEditor/getting-started.md:345]: Horizontal StackLayout

File: MAUI/ImageEditor/getting-started.md (hunk 6 above)
Comment (Shalini-Ashokan) [reply MAUI/ImageEditor/getting-started.md]: Changed

File: MAUI/ImageEditor/getting-started.md (hunk 7 above)
Comment (Shalini-Ashokan) [reply MAUI/ImageEditor/getting-started.md]: Changed

File: MAUI/ImageEditor/getting-started.md [hunk 8]
```diff
@@ -307,4 +307,97 @@ The image editor utilizes the `AspectFit` image scaling of Image control to fit
...
+    imageEditor.Source = ImageSource.FromResource("MyProject.Resources.Images.image.jpeg");
+    imageEditor.MinimumWidthRequest = 400;
+    horizontalLayout.Add(imageEditor);
+    this.Content = horizontalLayout;
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+### Image is edited or not.
```
Comment (karthikraja-arumugam) [inline MAUI/ImageEditor/getting-started.md:376]: move to above stack layout

**Review by florenceliliana — COMMENTED** [review]
⁠The content provided is fine.

File: MAUI/ImageEditor/getting-started.md [hunk 9]
```diff
@@ -307,4 +307,97 @@ The image editor utilizes the `AspectFit` image scaling of Image control to fit
...
+    if (this.imageEditor.IsImageEdited)
+    {
+        this.imageEditor.Save();
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+## ImageEditor inside stack layout
```
Comment (Jeyasri-Murugan) [inline MAUI/ImageEditor/getting-started.md:341]: Add the same in data form

File: MAUI/ImageEditor/getting-started.md [hunk 10]
```diff
@@ -307,4 +307,97 @@ The image editor utilizes the `AspectFit` image scaling of Image control to fit
 {% endhighlight %}
 {% endtabs %}
 
-N> The size value will only be available after the image has been loaded into view.
\ No newline at end of file
+N> The size value will only be available after the image has been loaded into view.
+
+### Image is edited or not.
```
Comment (Jeyasri-Murugan) [inline MAUI/ImageEditor/getting-started.md:312]: Check Image edited

File: MAUI/ImageEditor/getting-started.md [hunk 11]
```diff
@@ -307,4 +307,97 @@ The image editor utilizes the `AspectFit` image scaling of Image control to fit
 {% endhighlight %}
 {% endtabs %}
 
-N> The size value will only be available after the image has been loaded into view.
\ No newline at end of file
+N> The size value will only be available after the image has been loaded into view.
+
+### Check Image edited
```
Comment (florenceliliana) [inline MAUI/ImageEditor/getting-started.md:312]: ### Check the Edited Image

**Review by florenceliliana — CHANGES_REQUESTED** [review]

File: MAUI/ImageEditor/getting-started.md (hunk 11 above)
Comment (karthikraja-arumugam) [reply MAUI/ImageEditor/getting-started.md:312]: @florenceliliana the context here is **whether the image has been edited or not.**

Please suggest headline based on that.

File: MAUI/ImageEditor/getting-started.md (hunk 9 above)
Comment (Shalini-Ashokan) [reply MAUI/ImageEditor/getting-started.md:341]: Added in data form.
https://github.com/syncfusion-content/maui-docs/pull/1737

File: MAUI/ImageEditor/getting-started.md (hunk 10 above)
Comment (Shalini-Ashokan) [reply MAUI/ImageEditor/getting-started.md]: Updated

**Review by karthikraja-arumugam — APPROVED** [review]
This is fine

**Review by karthikraja-arumugam — APPROVED** [review]

**Pull request merged**